rust_snuba/src/processors/replays.rs

```diff
@@ -188,6 +188,15 @@ pub fn deserialize_message(
                 ..Default::default()
             }]
         }
+        ReplayPayload::ViewedEvent(event) => {
+            vec![ReplayRow {
+                replay_id: replay_message.replay_id,
+                viewed_by_id: event.user_id,
+                event_hash: Uuid::from_u64_pair(0, event.user_id), // for dedup
```

Comment:
Is this deterministic?

Reply:
https://docs.rs/uuid/latest/uuid/struct.Uuid.html#method.from_u64_pair
Yup, it just concatenates the inputs